src/pocketmine/level/Position.php

```diff
@@ -89,6 +89,62 @@ public function setLevel(Level $level = null){
 		return $this;
 	}
 
+	/**
+	 * @param Position|int $x
+	 * @param int         $y
+	 * @param int         $z
+	 *
+	 * @return Vector3
+	 */
+	public function add($x, $y = 0, $z = 0) : Position{
+		if($x instanceof Position){
```

Comment:
A Position object is a positional vector by definition. There is no reason to add two positional vectors.